azure/packages/azure-local-service/package.json

```diff
@@ -42,7 +42,7 @@
 		"@fluidframework/eslint-config-fluid": "^5.4.0",
 		"eslint": "~8.55.0",
 		"eslint-config-prettier": "~9.0.0",
-		"pm2": "^5.3.1",
+		"pm2": "^5.4.2",
```

Comment:
I thought we had found a replacement for pm2 - @alexvy86 @Josmithr does that ring any bells?

Reply:
I replaced `forever` with `pm2` in https://github.com/microsoft/FluidFramework/pull/19459. I remember some discussion after that at some point and IIRC my take then was "we probably don't _need_ pm2 where we use it, it's just a convenience; if anyone wants to find an alternative or just get rid of it, feel free". That said, I went through the legal review process it needed due to its licensing and we're ok to use it _as a dev dependency_, so we're not in urgent need of doing something about it.